src/app/pages/network/interfaces/interfaces-form/interfaces-form.component.ts

```diff
@@ -118,15 +117,15 @@ export class InterfacesFormComponent implements OnDestroy {
       formarray: [{
         name: 'alias_address',
         placeholder: T('IPv4 Address'),
-        tooltip: T('Enter a static IP address in the format\
-                    <i>###.###.###.###</i> if DHCP is unset.'),
+        tooltip: T('Enter a static IPv4 address. Example:\
```

Comment:
Do these fields disappear if DHCP is enabled?

Reply:
Correct. I requested this in ticket https://redmine.ixsystems.com/issues/31500 and hiding the proper fields when DHCP is enabled was added about 4 months.